docs/painless/painless-guide/painless-execute-script.asciidoc

```diff
@@ -164,3 +164,329 @@ Response:
   "result": 0.8
 }
 --------------------------------------------------
+
+==== Runtime field contexts
+
+Runtime field contexts execute a script used for a runtime field and return results in doc-values order.
+
+[[painless-execute-api-contexts-runtime-fields]]
+.Contexts
+[options="header"]
+|======
+| Name            | Emit Type      | Return Type
+| boolean_field   | boolean        | sorted list of booleans
+| date_field      | long (ms)      | sorted list of longs (ms)
+| double_field    | double         | sorted list of doubles
+| geo_point_field | long (encoded) | sorted list of longs (encoded)
+| ip_field        | String         | sorted list of BytesRef[] (encoded)
```

Comment:
I think `BytesRef` might not be a concept that folks are used to seeing outside of ES. The execute API returns the ips formatted as a string, right? So maybe that is what it should say?